ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py

```diff
@@ -835,3 +835,42 @@ def test_tensorflow_adjoint(
         on_device=on_device,
         matrix=x[0],
     )
+
+
+# qr
+@handle_frontend_test(
+    fn_tree="tensorflow.linalg.qr",
+    dtype_and_input=_get_dtype_and_matrix(),
+    test_with_out=st.just(False),
+)
+def test_tensorflow_qr(
+        *,
+        dtype_and_input,
+        frontend,
+        test_flags,
+        fn_tree,
+        on_device,
+):
+    input_dtype, x = dtype_and_input
+    ret, frontend_ret = helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        input=x[0],
+        test_values=False,
+    )
+    ret = [ivy.to_numpy(x) for x in ret]
```

Comment:
You dont need to add the logic to perform value testing here. This  should already be taken care of in the `helpers.test_frontend_function()` function